### ISSUE TITLE
Some Profiles Fail

### ISSUE THREAD

**Issue by emregeldegul** [bisguzar/twitter-scraper#95]
Hi
The profile function fails in some profiles. I made a few changes in the previous PR: [pr](https://github.com/bisguzar/twitter-scraper/pull/94)

However, the errors continue. Ex;

```
>>> profile = Profile('atifceylan')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/es/CODERLAB/twitter-scraper/twitter_scraper/modules/profile.py", line 30, in __init__
    self.__parse_profile(page)
  File "/home/es/CODERLAB/twitter-scraper/twitter_scraper/modules/profile.py", line 79, in __parse_profile
    self.tweets_count = int(q.attrs["title"].split(' ')[0].replace(',',''))
ValueError: invalid literal for int() with base 10: '8.393'
>>> 
```
I'll do my best.

**Comment by bisguzar** [comment]
Just fixed via #100, please try with never version and close issue if it success. Thanks for suggestion ^^

**Comment by emregeldegul** [comment]
Good job! this works :)
